Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -36,7 +37,7 @@ def set_marshall_helper(WorkspaceId, Enabled, AzureDnsAgentSvcZone, mock_worker_
     try:
         is_primary = is_oms_primary_workspace(WorkspaceId)
     except Exception, e:
-        log(ERROR, "Could not detect if workspace is primary\n %s" %(str(e)))
+        log(ERROR, "Could not detect if workspace is primary\n %s" % (str(e)))
```

Comment:
Why aren't these formatted like other python strings with variable insertions? I.E. `"Could not detect if workspace is primary\n " + (str(e))`

I'm not a python expert, so this may have an obvious answer, but I wanted to check.